src/Sulu/Bundle/CategoryBundle/Entity/Category.php

```diff
@@ -334,6 +334,22 @@ public function getTranslations()
     }
 
     /**
+     * Get single meta by locale.
+     *
+     * @param $locale
+     *
+     * @return CategoryTranslation
+     */
+    public function findTranslationByLocale($locale)
+    {
+        return $this->translations->filter(
+            function (CategoryTranslation $translation) use ($locale) {
+                return $translation->getLocale() === $locale;
```

Comment:
Is that really a performant way of doing that? Looks like all the available translations have to be hydrated...